src/GiraffeGenerator.Core/CodeGen.fs

```diff
@@ -236,38 +315,49 @@ let giraffeAst (api: Api) =
               }
               |> Map
 
-          let errorsPossible = hasErrorsPossible api
+          let validationErrorsPossible, nonValidationErrorsPossible = hasErrorsPossible api
           
           let allSchemas =
               [
-                if errorsPossible then
-                    yield! CodeGenErrorsDU.typeSchemas
+                if validationErrorsPossible || nonValidationErrorsPossible then
```

Comment:
I meant that either validation or non-validation errors are possible. But you're right: this condition would always be true because I decided to generate validation extension point even if there is no generated validation, i.e. there always may be a validation error.